src/app/grid/grid-editing-events/grid-editing-events.component.html

```diff
@@ -0,0 +1,10 @@
+<h4>Shipping Orders</h4>
+<igx-grid [data]="products" width="800px" height="500px" primaryKey="ProductID" (onCellEdit)="handleCellEdit($event)">
+    <igx-column field="ProductName" header="Product Name" dataType="string">
+    </igx-column>
+    <igx-column field="UnitPrice" header="Price" dataType="number"></igx-column>
+    <igx-column field="UnitsInStock" header="Units In Stock" dataType="number"></igx-column>
+    <igx-column field="Ordered" header="Ordered" dataType="number" [editable]="true">
```

Comment:
@ViktorSlavov  Can you make the Ordered column first? Really odd having to click through other to find out just this one is editable (despite the snippet in the docs)